.circleci/config.yml

```diff
@@ -0,0 +1,142 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@4.1.3
+jobs:
+  terraform_lint:
+    docker:
+      - image: quay.io/astronomer/ci-terraform:2020-12
+    steps:
+      - checkout
+      - run: cp providers.tf.example providers.tf
+      - run: terraform-0.12.29 init
+      - run: terraform-0.12.29 fmt -check=true
+      - run: terraform-0.12.29 validate -var "deployment_id=validate" -var "route53_domain=validate-fake.com" -var "admin_email=fake@mailinator.com"
+      - run: |
+            for example in $(find examples -maxdepth 1 -mindepth 1 -type d); do
+            cp providers.tf $example
+            cd $example
+            echo $example
+            terraform-0.12.29 init
+            terraform-0.12.29 fmt -check=true
+            terraform-0.12.29 validate -var "deployment_id=citest"
+            cd -
+            done
+      - run: terraform-0.12.29 -v
+      - slack/notify:
+          event: fail
+          template: basic_fail_1
+
+  terraform_apply:
+    docker:
+      - image: quay.io/astronomer/ci-terraform:2020-12
+    steps:
+      - checkout
+      - run: |
+            curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator
+            chmod +x ./aws-iam-authenticator
+            mv ./aws-iam-authenticator /usr/local/bin/
+            which aws-iam-authenticator
```

Comment:
optional: can this be added in the image quay.io/astronomer/ci-terraform:2020-12?

Reply:
Absolutely. Wherever possible we should avoid installing third party software in CI. I will make that change now.

Reply:
Done https://github.com/astronomer/ci-images/compare/268d78a1e..08dce78e

Reply:
Good catch @sjmiller609. @danielhoherd Thanks for adding, I'll give it a test run